kits/java/src/main/java/com/luxai/Bot.java

```diff
@@ -0,0 +1,77 @@
+package com.luxai;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Main class
+ * Lux-ai-2022
+ */
+public class Bot
+{
+    public static boolean DEBUG = false;
+    public static boolean REPLAY = false;
+
+    public static void main( String[] args ) throws IOException {
+        Agent agent = new Agent();
+        Scanner scanner = new Scanner(System.in);
+        if (!REPLAY) {     // Play
+            while (true) {
+                String json = scanner.nextLine();   // Read input
+                agent.updateState(json);            // Update state
+
+                if (DEBUG) {
+                    String fileIn = agent.me() + "_in.txt";
+                    FileWriter fw = new FileWriter(fileIn, true);
+                    fw.write(json + "\n");
+                    fw.close();
+                }
+
+                // - - - Main section - - -
+                String jsonAction = null;
+                if (agent.obs.real_env_steps < 0)
+                    jsonAction = agent.early_setup();
+                else {
+                    jsonAction = agent.act();
+                }
+                if (jsonAction == null) {
+                    ObjectMapper objectMapper = new ObjectMapper();
+                    jsonAction = objectMapper.createObjectNode().toString();
+                }
+                // ^ ^ ^ Main section ^ ^ ^
+
+                if (DEBUG) {
```

Comment:
make sure to document the debug tools you add here.

Reply:
fixed